### ISSUE TITLE
OpenShot-2.5.x wont run Debian 11 Bullseye

### ISSUE THREAD

**Issue by brcisna** [OpenShot/openshot-qt#3293]
Hello All,

 A fresh install of Debian 11 / Bullseye ,,only OpenShot-2.4.x will run . 
The versions including nightly builds of OS-2.5.x fails with the following.

The below terminal output only happens after trying to load a saved project,,,which I have many of,,all created with OS-2.4.x.

140469257038720:error:25066067:DSO support routines:DLFCN_LOAD:could not load the shared library:dso_dlfcn.c:185:filename(libssl_conf.so): libssl_conf.so: cannot open shared object file: No such file or directory
140469257038720:error:25070067:DSO support routines:DSO_load:could not load the shared library:dso_lib.c:244:
140469257038720:error:0E07506E:configuration file routines:MODULE_LOAD_DSO:error loading dso:conf_mod.c:285:module=ssl_conf, path=ssl_conf
140469257038720:error:0E076071:configuration file routines:MODULE_RUN:unknown module name:conf_mod.c:222:module=ssl_conf
QMutex: destroying locked mutex

They all end in the module=ssl_conf.

Not sure what that means.

This machine uses Intel HD graphics  but same results on a  machine with an old Matrix G200ex chip as well.

Thank You

**Comment by SuslikV** [comment]
I think your issue is variant of: https://github.com/OpenShot/openshot-qt/issues/3242
That itself has fix: https://github.com/OpenShot/openshot-qt/pull/3283

**Comment by ferdnyc** [comment]
Yeah, that reminds me, I need to merge that. It should be fairly harmless, except for affected systems.

**Comment by ferdnyc** [comment]
If you're interested, @brcisna , you can try downloading the next Daily Build AppImage package (which should be ready in a few minutes) from https://openshot.org/download/ and see if that does indeed fix the problem internally.

**Comment by ferdnyc** [comment]
(Please feel free to reopen if it _doesn't_.)

**Comment by ferdnyc** [comment]
My first fix (first two, actually) for this issue and #3242 was unsuccessful. However, the evidence is strong that #3305 solves this For Reals™, finally. That fix is included in the most recent AppImage Daily Builds.

**Comment by 0xcrypto** [comment]
Freshly downloaded AppImage of Openshot v2.5.1 is crashing as soon as a resource is added (by dragging or using the context menu) to the timeline with the error ```139664635353984:error:25066067:DSO support routines:DLFCN_LOAD:could not load the shared library:dso_dlfcn.c:185:filename(libssl_conf.so): libssl_conf.so: cannot open shared object file: No such file or directory```

Complete log: 
```
vi@vi:~$ /home/vi/Downloads/OpenShot-v2.5.1-x86_64.AppImage
zenity, kdialog, Xdialog missing. Skipping ./bin//openshot-qt.wrapper.
Loaded modules from current directory: /tmp/.mount_DKVbxV/usr/bin
         app:INFO ------------------------------------------------
         app:INFO             Sun May 10 08:39:14 2020            
         app:INFO               Starting new session              
         app:INFO ------------------------------------------------
         app:INFO             OpenShot (version 2.5.1)            
         app:INFO ------------------------------------------------
         app:INFO openshot-qt version: 2.5.1
         app:INFO libopenshot version: 0.2.5
         app:INFO platform: Linux-5.4.0-kali2-amd64-x86_64-with-debian-kali-rolling
         app:INFO processor: 
         app:INFO machine: x86_64
         app:INFO python version: 3.4.3
         app:INFO qt5 version: 5.2.1
         app:INFO pyqt5 version: 5.2.1
    language:INFO Qt Detected Languages: ['en-IN', 'en']
    language:INFO LANG Environment Variable: en_IN.utf8
    language:INFO LOCALE Environment Variable: 
    language:INFO OpenShot Preference Language: Default
project_data:INFO Setting default profile to HD 720p 30 fps
         app:INFO Setting font to /tmp/.mount_DKVbxV/usr/bin/images/fonts/Ubuntu-R.ttf
logger_libopenshot:INFO Connecting to libopenshot with debug port: 5556
         app:INFO Setting custom dark theme
QMainWindow::addDockWidget: invalid 'area' argument
     ui_util:INFO Initializing UI for MainWindow
 files_model:INFO updating files model.
transition_model:INFO updating transitions model.
effects_model:INFO updating effects model.
properties_model:INFO updating clip properties model.
transition_model:INFO updating transitions model.
 files_model:INFO updating files model.
 main_window:INFO InitCacheSettings
 main_window:INFO cache-mode: CacheMemory
 main_window:INFO cache-limit-mb: 250
 main_window:INFO Creating CacheMemory object with 262144000 byte limit
preview_thread:INFO QThread Start Method Invoked
 main_window:ERROR Unhandled crash detected... will attempt to recover backup project: /home/vi/.openshot_qt/backup.osp
preview_thread:INFO initPlayer
     version:INFO Found current version: {"openshot_version": "2.5.1"}
 main_window:INFO updateStatusChanged
 main_window:INFO updateStatusChanged
         app:INFO Process command-line arguments: ['/tmp/.mount_DKVbxV/usr/bin/openshot-qt']
 main_window:INFO recover_backup
project_data:INFO Setting default profile to HD 720p 30 fps
preview_thread:INFO refreshFrame
preview_thread:INFO self.player.Position(): 1
video_widget:INFO Load: Set video widget display aspect ratio to: 1.7777777910232544
video_widget:INFO Set video widget pixel aspect ratio to: 1.0
 main_window:INFO updateStatusChanged
 main_window:INFO foundCurrentVersion: Found the latest version: 2.5.1
preview_thread:INFO onModeChanged
properties_model:INFO Update item: 
properties_model:INFO updating clip properties model.
    timeline:INFO Adjusting max size of preview image: PyQt5.QtCore.QSize(724, 407)
preview_thread:INFO refreshFrame
preview_thread:INFO self.player.Position(): 1
preview_thread:INFO refreshFrame
preview_thread:INFO self.player.Position(): 1
timeline_webview:WARNING TimelineWebView::eval_js() called before document ready event. Script queued: $('body').scope().SetTrackLabel('Track %s');
timeline_webview:WARNING TimelineWebView::eval_js() called before document ready event. Script queued: $('body').scope().LoadJson({"old_values": {}, "partial": false, "value": {"display_ratio": {"den": 9, "num": 16}, "import_path": "", "profile": "HD 720p 30 fps", "settings": {}, "version": {"openshot-qt": "0.0.0", "libopenshot": "0.0.0"}, "scale": 15, "sample_rate": 44100, "pixel_ratio": {"den": 1, "num": 1}, "files": [], "channel_layout": 3, "layers": [{"number": 1000000, "label": "", "y": 0, "id": "L1", "lock": false}, {"number": 2000000, "label": "", "y": 0, "id": "L2", "lock": false}, {"number": 3000000, "label": "", "y": 0, "id": "L3", "lock": false}, {"number": 4000000, "label": "", "y": 0, "id": "L4", "lock": false}, {"number": 5000000, "label": "", "y": 0, "id": "L5", "lock": false}], "playhead_position": 0, "export_path": "", "channels": 2, "effects": [], "markers": [], "progress": [], "id": "WBBD5OEAWW", "tick_pixels": 100, "fps": {"den": 1, "num": 30}, "clips": [], "height": 720, "width": 1280, "duration": 300}, "type": "load", "key": ""});
timeline_webview:WARNING TimelineWebView::eval_js() called before document ready event. Script queued: $('body').scope().setScale(15, 0);
timeline_webview:WARNING TimelineWebView::eval_js() called before document ready event. Script queued: $('body').scope().SetSnappingMode(1);
timeline_webview:WARNING TimelineWebView::eval_js() called before document ready event. Script queued: $('body').scope().MovePlayheadToFrame(1);
timeline_webview:WARNING TimelineWebView::eval_js() called before document ready event. Script queued: $('body').scope().SetFollow(1);
timeline_webview:INFO Qt Found!
timeline_webview:INFO $scope.Qt = true;
timeline_webview:INFO SetThumbAddress: http://127.0.0.1:56205/thumbnails/
timeline_webview:INFO SortItems
timeline_webview:INFO UpdateLayerIndex
timeline_webview:INFO UpdateLayerIndex
project_data:INFO _set key: ['files'] values: {'display_ratio': {'den': 9, 'num': 16}, 'acodec': 'aac', 'vcodec': 'h264', 'path': '/home/vi/Downloads/poc.flv', 'channel_layout': 3, 'metadata': {'encoder': 'Lavf58.20.100'}, 'video_timebase': {'den': 1000, 'num': 1}, 'sample_rate': 44100, 'type': 'FFmpegReader', 'video_length': '4980', 'audio_timebase': {'den': 1000, 'num': 1}, 'id': 'GJFE6ZOMWF', 'pixel_format': 0, 'file_size': '2148904', 'has_single_image': False, 'video_bit_rate': 12888, 'channels': 2, 'audio_bit_rate': 128000, 'pixel_ratio': {'den': 1, 'num': 1}, 'media_type': 'video', 'has_audio': True, 'has_video': True, 'top_field_first': True, 'fps': {'den': 1, 'num': 30}, 'interlaced_frame': False, 'height': 540, 'video_stream_index': 0, 'duration': 166.0, 'audio_stream_index': 1, 'width': 960} add: True partial: False remove: False
timeline_webview:INFO SortItems
timeline_webview:INFO UpdateLayerIndex
 files_model:INFO updating files model.
[swscaler @ 0x7f05f43e02a0] Warning: data is not aligned! This can lead to a speed loss
 files_model:INFO updating files model.                                                                                                                                                                                                    
 main_window:INFO updateStatusChanged
 files_model:INFO updating files model.
project_data:INFO _set key: ['import_path'] values: /home/vi/Downloads add: False partial: False remove: False
 main_window:INFO updateStatusChanged
 main_window:INFO Imported media file /home/vi/Downloads/poc.flv
files_listview:INFO currentChanged
files_listview:INFO updateSelection
files_listview:INFO updateSelection
files_listview:INFO updateSelection
 main_window:INFO actionPreview_File_trigger
     ui_util:INFO Initializing UI for Dialog
     cutting:INFO /home/vi/Downloads/poc.flv
preview_thread:INFO QThread Start Method Invoked
preview_thread:INFO initPlayer
preview_thread:INFO onModeChanged
    timeline:INFO Adjusting max size of preview image: PyQt5.QtCore.QSize(562, 316)
preview_thread:INFO refreshFrame
preview_thread:INFO self.player.Position(): 1
     cutting:INFO sliderVideo_valueChanged
     cutting:INFO btnPlay_clicked
     cutting:INFO pause (icon to play)
preview_thread:INFO previewFrame: 2
preview_thread:INFO self.player.Position(): 2
     cutting:INFO sliderVideo_valueChanged
     cutting:INFO btnPlay_clicked
     cutting:INFO pause (icon to play)
preview_thread:INFO previewFrame: 1
preview_thread:INFO self.player.Position(): 1
     cutting:INFO btnPlay_clicked
     cutting:INFO play (icon to pause)
preview_thread:INFO onModeChanged
     cutting:INFO sliderVideo_valueChanged
     cutting:INFO btnPlay_clicked
     cutting:INFO pause (icon to play)
preview_thread:INFO previewFrame: 57
preview_thread:INFO self.player.Position(): 57
preview_thread:INFO onModeChanged
     cutting:INFO sliderVideo_valueChanged
     cutting:INFO btnPlay_clicked
     cutting:INFO pause (icon to play)
preview_thread:INFO previewFrame: 81
preview_thread:INFO self.player.Position(): 81
     cutting:INFO closeEvent
preview_thread:INFO exiting thread
preview_thread:INFO onModeChanged
project_data:INFO _set key: ['clips'] values: {'id': 'EFEBBE3K1R', 'perspective_c2_y': {'Points': [{'interpolation': 2, 'co': {'X': 1.0, 'Y': -1.0}}]}, 'volume': {'Points': [{'interpolation': 2, 'co': {'X': 1.0, 'Y': 1.0}}]}, 'location_y': {'Points': [{'interpolation': 2, 'co': {'X': 1.0, 'Y': 0.0}}]}, 'location_x': {'Points': [{'interpolation': 2, 'co': {'X': 1.0, 'Y': 0.0}}]}, 'anchor': 0, 'scale': 1, 'perspective_c2_x': {'Points': [{'interpolation': 2, 'co': {'X': 1.0, 'Y': -1.0}}]}, 'crop_x': {'Points': [{'interpolation': 2, 'co': {'X': 1.0, 'Y': 0.0}}]}, 'layer': 3000000, 'crop_height': {'Points': [{'interpolation': 2, 'co': {'X': 1.0, 'Y': 1.0}}]}, 'gravity': 4, 'perspective_c4_x': {'Points': [{'interpolation': 2, 'co': {'X': 1.0, 'Y': -1.0}}]}, 'scale_x': {'Points': [{'interpolation': 2, 'co': {'X': 1.0, 'Y': 1.0}}]}, 'alpha': {'Points': [{'interpolation': 2, 'co': {'X': 1.0, 'Y': 1.0}}]}, 'perspective_c3_y': {'Points': [{'interpolation': 2, 'co': {'X': 1.0, 'Y': -1.0}}]}, 'mixing': 0, 'rotation': {'Points': [{'interpolation': 2, 'co': {'X': 1.0, 'Y': 0.0}}]}, 'crop_width': {'Points': [{'interpolation': 2, 'co': {'X': 1.0, 'Y': 1.0}}]}, 'has_video': {'Points': [{'interpolation': 2, 'co': {'X': 1.0, 'Y': -1.0}}]}, 'perspective_c3_x': {'Points': [{'interpolation': 2, 'co': {'X': 1.0, 'Y': -1.0}}]}, 'scale_y': {'Points': [{'interpolation': 2, 'co': {'X': 1.0, 'Y': 1.0}}]}, 'shear_y': {'Points': [{'interpolation': 2, 'co': {'X': 1.0, 'Y': 0.0}}]}, 'reader': {'display_ratio': {'den': 9, 'num': 16}, 'acodec': 'aac', 'vcodec': 'h264', 'path': '/home/vi/Downloads/poc.flv', 'metadata': {'encoder': 'Lavf58.20.100'}, 'video_timebase': {'den': 1000, 'num': 1}, 'sample_rate': 44100, 'video_length': '4980', 'audio_timebase': {'den': 1000, 'num': 1}, 'video_stream_index': 0, 'pixel_format': 0, 'channel_layout': 3, 'has_single_image': False, 'video_bit_rate': 12888, 'channels': 2, 'audio_bit_rate': 128000, 'pixel_ratio': {'den': 1, 'num': 1}, 'type': 'FFmpegReader', 'has_audio': True, 'has_video': True, 'top_field_first': True, 'fps': {'den': 1, 'num': 30}, 'interlaced_frame': False, 'height': 540, 'file_size': '2148904', 'duration': 166.0, 'audio_stream_index': 1, 'width': 960}, 'position': 0.0, 'shear_x': {'Points': [{'interpolation': 2, 'co': {'X': 1.0, 'Y': 0.0}}]}, 'waveform': False, 'perspective_c4_y': {'Points': [{'interpolation': 2, 'co': {'X': 1.0, 'Y': -1.0}}]}, 'effects': [], 'crop_y': {'Points': [{'interpolation': 2, 'co': {'X': 1.0, 'Y': 0.0}}]}, 'title': 'poc.flv', 'time': {'Points': [{'interpolation': 2, 'co': {'X': 1.0, 'Y': 1.0}}]}, 'perspective_c1_x': {'Points': [{'interpolation': 2, 'co': {'X': 1.0, 'Y': -1.0}}]}, 'file_id': 'GJFE6ZOMWF', 'display': 0, 'end': 166.0, 'start': 0.0, 'channel_filter': {'Points': [{'interpolation': 2, 'co': {'X': 1.0, 'Y': -1.0}}]}, 'duration': 166.0, 'wave_color': {'blue': {'Points': [{'handle_left': {'X': 0.5, 'Y': 1.0}, 'interpolation': 0, 'handle_right': {'X': 0.5, 'Y': 0.0}, 'handle_type': 0, 'co': {'X': 1.0, 'Y': 255.0}}]}, 'alpha': {'Points': [{'handle_left': {'X': 0.5, 'Y': 1.0}, 'interpolation': 0, 'handle_right': {'X': 0.5, 'Y': 0.0}, 'handle_type': 0, 'co': {'X': 1.0, 'Y': 255.0}}]}, 'red': {'Points': [{'handle_left': {'X': 0.5, 'Y': 1.0}, 'interpolation': 0, 'handle_right': {'X': 0.5, 'Y': 0.0}, 'handle_type': 0, 'co': {'X': 1.0, 'Y': 0.0}}]}, 'green': {'Points': [{'handle_left': {'X': 0.5, 'Y': 1.0}, 'interpolation': 0, 'handle_right': {'X': 0.5, 'Y': 0.0}, 'handle_type': 0, 'co': {'X': 1.0, 'Y': 123.0}}]}}, 'has_audio': {'Points': [{'interpolation': 2, 'co': {'X': 1.0, 'Y': -1.0}}]}, 'channel_mapping': {'Points': [{'interpolation': 2, 'co': {'X': 1.0, 'Y': -1.0}}]}, 'perspective_c1_y': {'Points': [{'interpolation': 2, 'co': {'X': 1.0, 'Y': -1.0}}]}} add: True partial: False remove: False
timeline_webview:INFO SortItems
timeline_webview:INFO UpdateLayerIndex
Auto configuration failed
139664635353984:error:25066067:DSO support routines:DLFCN_LOAD:could not load the shared library:dso_dlfcn.c:185:filename(libssl_conf.so): libssl_conf.so: cannot open shared object file: No such file or directory
139664635353984:error:25070067:DSO support routines:DSO_load:could not load the shared library:dso_lib.c:244:
139664635353984:error:0E07506E:configuration file routines:MODULE_LOAD_DSO:error loading dso:conf_mod.c:285:module=ssl_conf, path=ssl_conf
139664635353984:error:0E076071:configuration file routines:MODULE_RUN:unknown module name:conf_mod.c:222:module=ssl_conf
QMutex: destroying locked mutex
vi@vi:~$ 
```

**Comment by XanderLeaDaren** [comment]
It seems it’s a Debian-side related [bug](https://bugs.debian.org/cgi-bin/pkgreport.cgi?dist=unstable;package=openshot-qt).

You may try adding the repository http://www.deb-multimedia.org/ to your sources.list — they have [version 2.5.1-dmo2](http://www.deb-multimedia.org/dists/testing/main/binary-i386/package/openshot-qt) for Bullseye.

I’m running Debian sid on a machine where OpenShot from deb-multimedia is launching fine. 👌🏽

**Comment by ferdnyc** [comment]
Or use the AppImage from the Daily Builds. The fix for this issue (#3304) went in after the release of 2.5.1.
